genie/input_to_database.py

```diff
@@ -747,28 +726,45 @@ def validation(
     return valid_filesdf[["id", "path", "fileType", "name"]]
 
 
+# TODO: probably should rename this for clarity
 def center_input_to_database(
-    syn,
-    project_id,
-    center,
-    process,
-    only_validate,
-    database_to_synid_mappingdf,
-    center_mapping_df,
-    delete_old=False,
-    oncotree_link=None,
-    genie_annotation_pkg=None,
-    format_registry=None,
+    syn: Synapse,
+    project_id: str,
+    center: str,
+    process: str,
+    only_validate: bool,
+    database_to_synid_mappingdf: pd.DataFrame,
+    delete_old: bool = False,
+    format_registry: list = None,
+    genie_config: dict = None,
 ):
+    """Processing per center
+
+    Args:
+        syn (Synapse): _description_
+        project_id (str): _description_
+        center (str): _description_
+        process (str): _description_
+        only_validate (bool): _description_
+        database_to_synid_mappingdf (pd.DataFrame): _description_
+        center_mapping_df (pd.DataFrame): _description_
+        delete_old (bool, optional): _description_. Defaults to False.
+        oncotree_link (str, optional): _description_. Defaults to None.
+        genie_annotation_pkg (str, optional): _description_. Defaults to None.
+        format_registry (typing.List, optional): _description_. Defaults to None.
+        genie_config (typing.Dict, optional): See example of genie config at
+                                              ./genie_config.json. Defaults to None.
```

Comment:
Where is the `genie_config.json` example located?  I couldn't find any json files in the repo

Reply:
It's not actually. I need to include an example but you can currently find it at tests/conftest.py.  (unless that's in the other PR as well)